sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusRetryPolicy.cs

```diff
@@ -139,22 +140,22 @@ public abstract class ServiceBusRetryPolicy
 
                     catch (Exception ex)
                     {
-                        Exception activeEx = AmqpExceptionHelper.TranslateException(ex);
+                        var activeEx = AmqpExceptionHelper.TranslateException(ExceptionDispatchInfo.Capture(ex));
```

Comment:
nit: we have a VS warning to use explicit types except when using literals or constructors.